src/Sulu/Bundle/WebsiteBundle/Controller/ExceptionController.php

```diff
@@ -35,64 +40,67 @@ class ExceptionController extends BaseExceptionController
      */
     private $parameterResolver;
 
-    /**
-     * @var ContentMapperInterface
-     */
-    private $contentMapper;
-
     public function __construct(
-        \Twig_Environment $twig,
+        EngineInterface $engine,
```

Comment:
Have we already removed our `EventAwareTwigEngine`? If this one is not used anymore some of this changes are not necessary, are they?